### PR TITLE
feat: add FAB size prop

### PR DESCRIPTION
**Pull request by joaoGabriel55**
Feature asked from issue #3064

### Summary

Add a prop size at FAB component

### Test plan

Final result (Updated preview):

image

### CODE REVIEW

**Comment by danieldanielecki** [comment]
Much awaited change as per https://github.com/callstack/react-native-paper/pull/921

**Comment by lukewalczak** [comment]
Hey @joaoGabriel55, due to renaming `next` branch (with Material You changes) to `main` there are conflicts. Could you please resolve it? If you'll need some help, please let me know.

**Comment by joaoGabriel55** [comment]
> Hey @joaoGabriel55, due to renaming `next` branch (with Material You changes) to `main` there are conflicts. Could you please resolve it? If you'll need some help, please let me know.

Yeah. I did rebase with the `main` branch, that is it?

File: src/components/FAB/FAB.tsx [hunk 1]
```diff
@@ -86,6 +86,10 @@ type Props = $RemoveChildren<typeof Surface> & {
    * - `large` - Appbar with large height (96).
    */
   size?: FABSize;
+  /**
+   * Custom size for the `FAB`.
```
Comment (lukewalczak) [inline src/components/FAB/FAB.tsx:90]: I would add a note that prop takes precedence over `size` prop.

**Review by lukewalczak — CHANGES_REQUESTED** [review]

File: src/components/FAB/FAB.tsx (hunk 1 above)
Comment (lukewalczak) [reply src/components/FAB/FAB.tsx:90]: Please also add an annotation that prop is supported in v5:
`   * @supported Available in v5.x`

**Pull request merged**